admin/jobs/__init__.py

```diff
@@ -21,61 +17,6 @@ def index():
     return ('', 200)
 
 
-@jobs.route('/jobs/check-user-input-formats', methods=['GET'])
-@JobTokenRequired
-def check_user_input_formats():
-    number_of_applications_in_old_format: int = 0
-    number_of_applications_in_new_format: int = 0
-
-    applications = Application.query.all()
-
-    for application in applications:
-        try:
-            application_data_obj: ApplicationData = jsonpickle.decode(application.user_input)
-            number_of_applications_in_new_format = number_of_applications_in_new_format + 1
-        except:
-            number_of_applications_in_old_format = number_of_applications_in_old_format + 1
-
-    return f"{number_of_applications_in_old_format} applications in old format. {number_of_applications_in_new_format} applications in new format"
-
-
-
-@jobs.route('/jobs/migrate-user-input-to-new-format', methods=['GET'])
-@JobTokenRequired
-def migrate_user_input_to_pickle_format():
-    number_of_applications_in_old_format: int = 0
-    number_of_applications_in_new_format: int = 0
-    number_of_applications_with_no_data: int = 0
-    data_migration_failed_for_these_applications = []
-
-    applications = Application.query.all()
-
-    for application in applications:
-        try:
-            if application.user_input:
-                application_data_obj: ApplicationData = jsonpickle.decode(application.user_input)
-                number_of_applications_in_new_format = number_of_applications_in_new_format + 1
-            else:
-                application_data_obj: ApplicationData = ApplicationData()
-                number_of_applications_with_no_data = number_of_applications_with_no_data + 1
-                application.user_input = jsonpickle.encode(application_data_obj)
-        except:
-            try:
-                application_data_obj: ApplicationData = convert_weakly_typed_to_strongly_typed(ast.literal_eval(application.user_input))
-                number_of_applications_in_old_format = number_of_applications_in_old_format + 1
-                application.user_input = jsonpickle.encode(application_data_obj)
-            except:
-                data_migration_failed_for_these_applications.append(application.reference_number)
-
-    db.session.commit()
-
-    return f"Conversion complete." \
-           f" {number_of_applications_in_old_format} converted from old to new format." \
-           f" {number_of_applications_in_new_format} already in new format." \
-           f" Migration failed for {data_migration_failed_for_these_applications}"
-
-
-
```

Comment:
I've run the migration on all our environments and all rows have been migrated, so we don't need this any more